src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -27,17 +26,51 @@ const ActorInit Bg_Inibs_Movebg_InitVars = {
     (ActorFunc)BgInibsMovebg_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96578[] = {
+s32 D_80B96560[] = { 0x060062D8, 0x06001DC0 };
+s32 D_80B96568[] = { 0x06006140, 0x06001C10 };
+s32 D_80B96570[] = { 0x06006858, 0x06002598 };
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
 };
 
-#endif
+void BgInibsMovebg_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgInibsMovebg* this = (BgInibsMovebg*)thisx;
+    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
+    DynaPolyActor_Init(&this->dyna, 1);
+
+    //for some reason doesn't match if I don't use the THIS macro twice here
+    THIS->unk_15C = D_80B96560[BGINIBSMOVEBG_GET_F(this->dyna)];
+    this->unk_160 = D_80B96568[BGINIBSMOVEBG_GET_F(this->dyna)];
+    this->unk_164 = D_80B96570[BGINIBSMOVEBG_GET_F(THIS->dyna)];
+}
+
+void BgInibsMovebg_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    DynaPoly_DeleteBgActor(globalCtx, &globalCtx->colCtx.dyna, THIS->dyna.bgId);
+}
```

Comment:
```suggestion
void BgInibsMovebg_Destroy(Actor* thisx, GlobalContext* globalCtx) {
    BgInibsMovebg* this = THIS;

    DynaPoly_DeleteBgActor(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
}
```
Generally always cast with `THIS` then just use `this`, unless of course it does not match.